capabilityreceiver/pkg/chplmapper/capabilityinfo.go

```diff
@@ -79,6 +102,38 @@ func getVendorMatch(ctx context.Context, capStat capabilityparser.CapabilityStat
 	return vendorID, nil
 }
 
+func openProductLinksFile(filepath string) (map[string]map[string]string, error) {
+	jsonFile, err := os.Open(filepath)
+	if err != nil {
+		return nil, err
+	}
+	defer jsonFile.Close()
+
+	var softwareNameVersion []map[string]string
+	byteValueFile, err := ioutil.ReadAll(jsonFile)
+	if err != nil {
+		return nil, err
+	}
+	var chplMap = make(map[string]map[string]string)
+	if len(byteValueFile) != 0 {
+		err = json.Unmarshal(byteValueFile, &softwareNameVersion)
+		if err != nil {
+			return nil, err
+		}
+		for _, obj := range softwareNameVersion {
+			var name = obj["name"]
```

Comment:
I think it might make sense to have some sort of check here to verify that name, version, and CHPLID exists, just someone messes up while entering a new value in the file ("CHLPID" instead of "CHPLID" for example) it seems like that should be handled gracefully

Reply:
I think it probably makes sense to also have a test for this function if the above change is made